### ISSUE TITLE
Blogger starter kit ? 

### ISSUE THREAD

**Issue by o0Ignition0o** [rust-community/content-o-tron#42]
TL;DR : Could we consider a "new blogger starter kit ?" This would probably include how to setup gh-pages or jekyll or any markdown supporting platform, how to get it hosted, get a domain name etc.

As part of trying to help newcomers doing their first Pull Request during the Outreachy project, we often had to explain how to work with github (how to rebase / squash commits, how to handle conflicts etc.). https://opensource.guide does an amazing job at providing an open source starter kit.

As a rookie blogger, I could be having lots of troubles "setting up" my blog : 
Should I go to Medium ? Should I rather publish markdown articles on gh-pages ? What is markdown and how am I supposed to work with it ? Can I embed links ? How could I promote my first articles ?

A resource on how to setup a blog to write and publish content could prove useful. 

What do you think? Would it be a good idea, or is it a bit off topic since this project seems to rather focus on "post content" than on "meta post" things.

I might not be the most suited person to provide starter kit contents, but I'd sure love to help (and maybe I could blog about the process later ;) )

**Comment by skade** [comment]
Reflecting my chat comments: I really like that and people have used the Rust blog as a template informally before. A forkable template with setup instructions would be great!

We can consider moving this to the @rust-community/content-team repository or consider it part of content-o-tron.

**Comment by booyaa** [comment]
This is a great idea!  

I would agree with @skade that this better suited as either part of the content-team repo or its own repo. 

Once it’s been reviewed we could then transfer into the Rust community org?

**Comment by adityac8** [comment]
Hey @o0Ignition0o You could have a look at [this blog](https://channelcs.github.io/best-practices-in-a-collaborative-environment.html). It is written by me and one of my friend who got into Outreachy. We can surely discuss more about it.

**Comment by o0Ignition0o** [comment]
I was actually looking for this link and couldn't find it anymore. Thanks a lot ! :D

**Comment by o0Ignition0o** [comment]
Ok so I have created a [github-page](https://o0ignition0o.github.io/rust-lang-blog-boilerplate/) with a sample [blogpost](https://o0ignition0o.github.io/rust-lang-blog-boilerplate/2018/05/10/blog-boilerplate.html) and hosted the source code on [a repo](https://github.com/o0Ignition0o/rust-lang-blog-boilerplate). 

I've even been able to submit a PR to fix a CVE in the [rustlang blog](https://github.com/rust-lang/blog.rust-lang.org/pull/247) \o/ 

I plan on creating my own blog to explain the steps in order to setup the whole environment.

Should I continue here or move it all to the content-team repo ?

**Comment by o0Ignition0o** [comment]
http://blog.jeremylempereur.com/2018/05/10/blogception-create-blog-jekyll-github-pages-part-1.html

Here's the first part of a series which explains how to setup a blog from the boilerplate, I hope I'll be able to write part two (editing _config.yml and writing the first post) tonight :)

**Comment by booyaa** [comment]
@o0Ignition0o nice work! thanks for that rust-www (CVE) too! 

I'm going to create raise with the main community about transferring repo to our org (as it implies ownership and on-going maintainance).

Please continue your blog post series, as I think I'm going to create another guide about setting up a blog and reference your posts (as well as the template repo).

**Comment by o0Ignition0o** [comment]
https://blog.jeremylempereur.com/2018/05/11/blogception-create-blog-jekyll-github-pages-part-2.html is out.

Editing the _config.yml file, and writing the first blog post :) 

Step 3 will probably come next week, and will probably include setting up gh pages, and using a custom domain name.

If you have any ideas on how to improve Section number 3 (to reflect more on the work that is being done here) please let me know, I would love to make it more inspiring !

**Comment by booyaa** [comment]
Discuss this with the whole community team, we're happy for you to transfer the ownership of the repo if you still want to do it. Just let us know when it's been done. Thanks!

**Comment by o0Ignition0o** [comment]
Sure, I'll do it by the end of the week \o/ 

Btw are there any things I should change before transferring ownership ?
I could also maintain it a bit if need be :)

**Comment by booyaa** [comment]
We'll make sure you're still a collaborator/maintainer when it's transferred! :)

**Comment by o0Ignition0o** [comment]
Ok so i just tried to transfer it and I got an error 

 `You don’t have the permission to create repositories on rust-community `

**Comment by o0Ignition0o** [comment]
@booyaa do you have any idea on how I can transfer ownership of the repository ?

**Comment by badboy** [comment]
@o0Ignition0o Looking into that now

**Comment by badboy** [comment]
@o0Ignition0o See https://help.github.com/articles/transferring-a-repository-owned-by-your-personal-account/#transferring-to-an-organization

Solution: if you transfer that repository to my account I will then transfer it into the organization and make sure you're added as an collaborator.

**Comment by o0Ignition0o** [comment]
Transfer ownership

This repository is being transferred to @badboy. 

SGTM :D

**Comment by badboy** [comment]
The repository is now available in the rust-community orga: https://github.com/rust-community/rust-lang-blog-boilerplate

The community team members, content team members as well as @o0Ignition0o especially do have write access to it.
